client/cli/ok.py

```diff
@@ -92,6 +93,12 @@ def parse_input(command_input=None):
 
 def main():
     """Run all relevant aspects of ok.py."""
+
+    #Checking user's Python bit version
+    bit_v = (8 * struct.calcsize("P"))
+    if (bit_v == 32):
+        print("32 bit Python is not supported. Please install the 64 bit version.")
```

Comment:
log.setlevel tells the logger to only emit log messages that are at or
above a certain level. We already set the log level in main. I just
included it in my comment to show that you should put your code under the
line that sets the log level.

On Sat, Jan 23, 2016, 6:55 AM Hayden Sheung <notifications@github.com>
wrote:

> In client/cli/ok.py
> <https://github.com/Cal-CS-61A-Staff/ok-client/pull/153#discussion_r50620899>
> :
>
> > @@ -92,6 +93,12 @@ def parse_input(command_input=None):
> >
> >  def main():
> >      """Run all relevant aspects of ok.py."""
> > +
> > +    #Checking user's Python bit version
> > +    bit_v = (8 * struct.calcsize("P"))
> > +    if (bit_v == 32):
> > +        print("32 bit Python is not supported. Please install the 64 bit version.")
>
> I thought I added a comment here but apparently Github didn't get it --
> what does setlevel() does?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Cal-CS-61A-Staff/ok-client/pull/153/files#r50620899>.
>